source/sync/data-access-patterns/permissions.txt

```diff
@@ -489,3 +653,26 @@ A user with write permissions for a given collection can modify the value of any
 field of any object matching the client-side query. Write permissions require
 read permissions, so any user that can modify data can also view that data
 before and after it's modified.
+
+.. _flexible-sync-field-level-permissions:
+
+Field-Level Permissions
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can refine document-level permissions on a per-field basis with the
+``fields`` and ``additional_fields`` properties of the **Role** configuration
+object.
+
+If the user does not have document-level permissions to a document, field-level
```

Comment:
Is this worth putting in a note to make it stand out a bit more? It feels like a "gotcha"